packages/shared/styles/components/atoms/SfImage.scss

```diff
@@ -1,52 +1,48 @@
 @import "../../helpers";
-.sf-image {
+
+.sf-image--wrapper {
   position: relative;
   display: inline-block;
-  line-height: 0;
-  overflow: hidden;
-  img {
-    width: var(--image-width);
-    max-width: 100%;
-    height: var(--image-height, auto);
+
+  &:hover{	
+    --image-overlay-opacity: 1	
   }
-  &.has-size {
-    width: 100%;
-    max-width: calc(var(--_image-width) * 1px);
-    img:not(.noscript) {
-      position: absolute;
-      top: 50%;
-      left: 0;
-      transform: translate3d(0, -50%, 0);
-    }
-    &::after {
-      display: block;
-      content: "";
-      padding-bottom: calc(var(--_image-height) / var(--_image-width) * 100%);
-    }
-  }
-  &__overlay {
+}
+
+.sf-image{    
+  object-fit: cover;
+  width: var(--image-width);
+  height: var(--image-height, auto);
+  display: flex;
+
+  &:not(.sf-image-loaded) {
+    opacity: 0;
     position: absolute;
-    top: 0;
-    right: 0;
-    left: 0;
-    bottom: 0;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    padding: var(--image-overlay-padding, var(--spacer-sm));
-    background: var(--image-overlay-background, rgba(var(--c-dark-base), 0.6));
-    color: var(--image-overlay-color, var(--c-white));
-    opacity: var(--image-overlay-opacity, 0);
-    @include font(
-      --image-overlay-font,
-      var(--font-weight--light),
-      var(--font-size--sm),
-      1.6,
-      var(--font-family--secondary)
-    );
-    transition: opacity 150ms ease-in-out;
   }
-  &:hover{
-    --image-overlay-opacity: 1
+
+  &:not([alt]),
+  &[alt=""] {
+    outline-color: red;
+    outline-width: 1px;
+    outline-style: dashed;
   }
 }
+
+.sf-image--placeholder {
+  display: inline-block;
+}
+
+.sf-image--overlay {
```

Comment:
what do you think about adding also font properties here? We have an example of an image with an overlay on figma :)

Reply:
I think font should be decided by user rather by us. It's more like a customized think. But we can use `font-family: --c-font` (or something default)